foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -179,4 +177,4 @@ This section contains questions for you to check your understanding of this less
 - <a class="knowledge-check-link" href='#em-element'>What element should you use to make text italicized to add emphasis to it?</a>
 - <a class="knowledge-check-link" href='#nested-relationship'>What relationship does an element have with any nested elements within it?</a>
 - <a class="knowledge-check-link" href='#elements-same-level'>What relationship do two elements have if they are at the same level of nesting?</a>
-- <a class="knowledge-check-link" href='#html-comments'>How do you create HTML comments?</a>
\ No newline at end of file
+- <a class="knowledge-check-link" href='#html-comments'>How do you create HTML comments?</a>
```

Comment:
Can you also add a newline at the end of the file?

Reply:
okay, i'll add a new line, or do you want me to use a <br /> tag instead

Reply:
Just an empty line.

Reply:
okay!!

Reply:
btw, i forgot to create a branch for this, i'll have to create new pull request!!

i'll try to be more mindful about this from now, thanks